rero_ils/modules/items/api/circulation.py

```diff
@@ -185,6 +185,30 @@ class ItemCirculation(IlsRecord):
         LoanState.ITEM_IN_TRANSIT_TO_HOUSE: 'in_transit',
     }
 
+    def process_item_and_loans(self, item):
+        """Commit and reindex the item and its loans."""
```

Comment:
L.73 it's not easy to understand what you do with this method unless you read the code.
The title of method just say "I do something". And the comment say: "I commit changes and reindex items and loans".

What about something like `commit_and_reindex` or `commit_and_reindex_item_and_loan`? Or `commit_and_reindex_with_loans`?